src/rust/grapl-observability/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "grapl-observability"
+version = "0.0.1"
+authors = ["Max Wittek <wimax@graplsecurity.com>"]
+edition = "2018"
+description = "Metrics and logging tooling for Grapl"
+license = "Apache-2.0"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+log = "0.4.*"
+regex = "1.1.0"
+lazy_static = "1.2.0"
+failure = "0.1.*"
```

Comment:
failure is deprecated. Since this is a library you should just roll your own error type instead.

Check this out.
https://github.com/dtolnay/thiserror